File: map/Interpreter2.0/model/ConstExpresion.cs
using DataStructure;
using System;
namespace models
{
    public class ConstExpresion : Expression
    {
        int val;

        public ConstExpresion(int value){
            val = value;
        }


        public override int eval(MyIntDict<String, int> tab) {
            return val;
        }

        override public String ToString() {
            return this.val.ToString();
        }
        
    }
}

File: map/InterpreterFX/src/model/ConstExpresion.java
package model;

import DataStructure.IntHeap;
import DataStructure.MyIntDict;
import ExceptionHandling.MyException;

public class ConstExpresion extends Expresion{

    int val;

    public ConstExpresion(int value){
        val = value;
    }

    @Override
    int eval(MyIntDict<String, Integer> tab, IntHeap heap) throws MyException {
        return val;
    }

    @Override
    public String toString() {
        return Integer.toString(this.val);
    }
}
